html/index.html

```diff
@@ -14,10 +14,6 @@
         <a href="fetch-loader-demo.html">FetchLoader (retrieves data via <code>fetch</code>)</a>
       </li>
 
-      <li>
-        <a href="full-demo.html">FetchLoader (retrieves real data from <code>localhost:8080/gxa/sc</code>)</a>
-      </li>
-
```

Comment:
This example is useful to test against running instances of SCEA. You probably forgot to put it back, could you add the link again and test that it works?